src/nunjucks/moduk/components/radios/__examples__/small.njk

```diff
@@ -0,0 +1,31 @@
+{% from "moduk/components/radios/macro.njk" import modukRadios %}
+
+{{ modukRadios({
+  name: "organisation",
+  classes: "govuk-radios--small",
+  fieldset: {
+    legend: {
+      text: "Organisation",
```

Comment:
(This one should stay as Organisation, since the small version is meant for filters.)